sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -393,6 +397,8 @@ def initialize
       self.traces_sampler = nil
       self.enable_tracing = nil
 
+      self.profiler = Sentry::Profiler
```

Comment:
Can we name this `profiler_class` instead?

Reply:
Good point! Renamed.